interpretation/train_classifier.py

```diff
@@ -0,0 +1,219 @@
+import pandas as pd
+import os
+from matplotlib import pyplot as plt
+import unicodedata
+from interpretation.plotting import get_roc_df, roc_plot_grid, thr_plot_grid
+from interpretation.report_reader import slice_df, normalize_od, offset_od
+from sklearn import metrics
+from sklearn.model_selection import GridSearchCV, GroupKFold, GroupShuffleSplit
+from xgboost.sklearn import XGBClassifier
+import xgboost as xgb
+#%%
+def xgb_fit(model, dtrain, features, target, cv=True, folds=None, cv_folds=5, early_stopping_rounds=20):
+    """Train xgboost model
+    :param object model: XGBClassifier instance
+    :param dataframe dtrain: training data with rows being samples and columns being features
+    :param list features: column names of features
+    :param str target: column name of the target
+    :param bool cv: if true, optimal number of estimators is determined by cross-validation
```

Comment:
Here I'm trying to be consistent with the scikit-learn classifier parameter convention. opencv is almost never passed as a function argument so I think this should be ok?

Reply:
Who knows, you might be ok bug-wise... but since cv is used for a different purpose elsewhere in this repo one could argue that you should be consistent with pysero conventions first and other module conventions second.

Reply:
Good point on naming convention consistency throughout the repo. I've changed `cv` to `cross_valid`